usagereportsv4/usage_reports_v4_integration_test.go

```diff
@@ -118,22 +96,15 @@ var _ = Describe(`UsageReportsV4 Integration Tests`, func() {
 			shouldSkipTest()
 		})
 		It(`GetAccountSummary(getAccountSummaryOptions *GetAccountSummaryOptions)`, func() {
-
 			getAccountSummaryOptions := &usagereportsv4.GetAccountSummaryOptions{
-				AccountID:    &accountID,
-				Billingmonth: &billingMonth,
+				AccountID: core.StringPtr("testString"),
```

Comment:
You have re-generated the integration tests and overwritten the code that was there previously, including manually developed code.  Is this really what you want?
You can see from this set of diffs that this test block previously used `accountId` and `billingMonth` which are variables that are obtained from the .env configuration file.